singer_encodings/csv_helper.py

```diff
@@ -0,0 +1,147 @@
+import csv
+import json
+import singer
```

Comment:
Upon further discussion, it seems best to avoid any possible dependency issues and remove the logging and not add a `singer-python` dependency. The tap can be in charge of logging any extra keys that are found.

Reply:
I don't feel particularly strongly either way, except for the concern about potential conflicts in `singer-python` version. As long as these log lines would be useful in debugging. One note regarding that is that the `debug` level of logging is disabled by default, so for traceability through historical logs, that log line should probably be `info` level.

Re: Adding the `singer-python` dependency to `setup.py`, as long as it doesn't specify a particular version then I am more comfortable with including it.

Reply:
Added singer-python in setup.py without version name.